Winch/Patches/API/GameSceneInitializerPatcher.cs

```diff
@@ -0,0 +1,18 @@
+﻿using HarmonyLib;
+using Winch.Core.API;
+
+namespace Randomizer.Patchers;
```

Comment:
I'm aware this is from a while ago, but just looked and spotted this 😅 
Should be this as far as I can see?
```suggestion
namespace Winch.Patches.API;
```